demo/using_decorators.ts

```diff
@@ -0,0 +1,33 @@
+import { accessorDecorator, annotatedAccessorDecorator, classDecorator, methodDecorator } from "./decorators";
+
+@classDecorator
+export class Person {
+    private name_: string;
+    constructor(name: string) {
+        this.name_ = name;
+    }
+
+    @accessorDecorator(true)
+    get name() {
+        return this.name_;
+    }
+
+    @annotatedAccessorDecorator(true)
+    get age() {
```

Comment:
In the demo this is getting emitted as a normal decorator, not converted to an annotation.

Reply:
Huh, when I followed the testing directions from the other pr I get this output:

```ts
__decorate([
    annotatedAccessorDecorator(true)
], Person.prototype, "age", null);
```

I also just noticed `__decorate` is defined inline instead of being imported too, which doesn't seem right.

Reply:
What??? That doesn't make any sense :(